driver/ssh/config.go

```diff
@@ -47,13 +52,15 @@ func (conf *Config) DecodeFromDSN(encodedDSN string) (dsn string, err error) {
 
 	parsed.RawQuery = values.Encode()
 
-	// remove the middle information of
-	// scheme://ssh_user:ssh_password@ssh_host:ssh_port/
+	// remove the middle information of scheme://ssh_user:ssh_password@ssh_host:ssh_port/
 	splitted := strings.Split(parsed.String(), "://")
```

Comment:
Split is an irregular verb. Can we rename this in `split` since `splitted` doesn't make sense in English?